.gitignore

```diff
@@ -53,4 +53,8 @@ bin/tmp/*
 scratchpad
 gen.gen
 
-**/*-wal/
\ No newline at end of file
+**/*-wal/
+.devcontainer
+/eudico-stats
+f1*
+Dockerfile.eudico
```

Comment:
Why do we ignore this? I don't expect anyone to have this file locally (except maybe you in your working environment). I'd rather remove this from the .gitignore.